server/src/Core/Interfaces/IGameNameDuplicationsChecker.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+
+namespace BGS.ApplicationCore.Interfaces;
+
+public interface IGameNameDuplicationsChecker
+{
+    Task<bool> Check(string gameName, CancellationToken cancellationToken);
```

Comment:
Pass the cancellationToken is optional, in this case I think we can leave it out

Reply:
removed